test/eoftest/eoftest_runner.cpp

```diff
@@ -22,23 +22,39 @@ struct EOFValidationTest
     {
         struct Expectation
         {
-            evmc_revision rev;
-            bool result;
+            evmc_revision rev = EVMC_PRAGUE;
+            bool result = false;
         };
         std::string name;
         evmc::bytes code;
+        ContainerKind kind = ContainerKind::runtime;
         std::vector<Expectation> expectations;
     };
     std::unordered_map<std::string, Case> cases;
 };
 
+ContainerKind container_kind_from_string(std::string_view s)
+{
+    if (s == "runtime")
+        return ContainerKind::runtime;
+    else if (s == "initcode")
+        return ContainerKind::initcode;
+    else if (s == "initcode_runtime")
+        return ContainerKind::initcode_runtime;
+    else
+        throw std::invalid_argument{"unknown container kind"};
+}
+
 void from_json(const json::json& j, EOFValidationTest::Case& o)
 {
     const auto op_code = evmc::from_hex(j.at("code").get<std::string>());
     if (!op_code)
         throw std::invalid_argument{"code is invalid hex string"};
     o.code = *op_code;
 
+    if (const auto it_kind = j.find("kind"); it_kind != j.end())
```

Comment:
Is this standardized?

Reply:
No, this is not supported anywhere, but I plan to try to add this to EEST.